openwisp_monitoring/notifications/migrations/0005_auto_20200421_2201.py

```diff
@@ -0,0 +1,16 @@
+# Generated by Django 3.0.4 on 2020-04-21 20:01
```

Comment:
watch out for the name of the file, we don't allow the default auto-generated names, openwisp-qa-check will complain